<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="styleSheet">
      <string notr="true">font: 27pt &quot;Rubik SemiBold&quot;;
color: rgb(0, 51, 102);
background-color: rgba(255, 255, 255, 0);
</string>
     </property>
     <property name="text">
      <string>Questionario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="question">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Rubik light&quot;;
color: rgb(53, 53, 53);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Domanda</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="answer_layout">
     <item>
      <widget class="QRadioButton" name="scale_1">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Rubik Light&quot;;</string>
       </property>
       <property name="text">
        <string>Per niente</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scale_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Rubik Light&quot;;</string>
       </property>
       <property name="text">
        <string>Poco</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scale_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Rubik Light&quot;;</string>
       </property>
       <property name="text">
        <string>Abbastanza</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scale_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Rubik Light&quot;;</string>
       </property>
       <property name="text">
        <string>Molto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scale_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Rubik Light&quot;;</string>
       </property>
       <property name="text">
        <string>Moltissimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="error_text">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Rubik Light&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255,0,0);</string>
       </property>
       <property name="text">
        <string>Per favore scegli una risposta.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
font: 15pt &quot;Rubik Light&quot;;
color: rgb(255,255,255);
background-color: rgb(0, 64, 130);</string>
       </property>
       <property name="text">
        <string>Avanti</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
